docs/exceptions.rst

```diff
@@ -37,3 +37,9 @@ Exceptions
 
     This is raised when the verify method of a key derivation function's
     computed key does not match the expected key.
+
+
+.. class:: InvalidTag
```

Comment:
exceptions.rst is indeed horrible. I'd suggest we should open an issue to figure out whether we can find better homes for everything in `cryptography.exceptions`. That seems like a larger task than just making sure we actually have SOMETHING written down about it (note that we don't even document that finalize/finalize_with_tag can raise InvalidTag except as a comment in the doctest)